incubator-tools/date_entities_annotation_tool/date_entities_annotation.ipynb

```diff
@@ -0,0 +1,567 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #206.        def create_entities(final_match,page,json_dict):
</pre><p>This whole function is difficult to read/understand.</p><p> </p><p>It would be easier if the new entities are initialized with all of the values instead of assigning the fields later.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/682/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>